offchain/danaswap-ui/views/components/head.ejs

```diff
@@ -0,0 +1,22 @@
+<meta charset="utf-8" />
+<meta http-equiv="X-XSS-Protection" content="1;mode=block" />
+<meta http-equiv="X-Content-Type-Options" content="nosniff" />
+<meta http-equiv="Permissions-Policy" content="interest-cohort=()" />
+<meta http-equiv="Referrer-Policy" content="strict-origin-when-cross-origin" />
```

Comment:
We still want a CSP now rather than later. I see no reason given the simplicity of this site that this CSP should not work:

```html
<!-- While we _should_ prefer default-src 'none', Fx has a bug with <use href>—thus we need 'self' -->
<meta http-equiv = "Content-Security-Policy" content = "default-src 'self'; img-src 'self'; object-src 'none'; script-src 'self'">
```

And everything that fails this CSP should be considered a bug.

Reply:
here're the errors after adding CSP

![image](https://user-images.githubusercontent.com/48898/199082817-4d2c0bc3-cb49-47f0-8053-f6a41f67cd94.png)

Reply:
Ah yes. We would need a flag for `IS_DEVELOPMENT` because the Webpack dev server can't function without `unsafe-eval`. The CSP error about Font Awesome from a third party is what we *want* to see :)

Reply:
I see this error `Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".` even if I change the `mode` in webpack to `development`

Reply:
I meant in the template like `<% if DEVELOPMENT_MODE %> ....`

Reply:
I am trying to introduce a new package for dev only 7f9619f2e6af0eab7951aab8ef1790d36869f73d, let's see if it will help the case

Reply:
@toastal now the production is down https://6363180057449c006f2e55f1--danaswap-ui.netlify.app/

Reply:
I reckon the only option I've got is to allow `unsafe-eval` 
eff52246aecf16ad27cb38cb66895f5e67df7077

Reply:
now it's working in prod https://6363217b695f0a06f6540419--danaswap-ui.netlify.app/

Reply:
and since we allow `unsafe-eval`, we don't need `IS_DEVELOPMENT` in template

Reply:
Well, allowing `unsafe-eval` should be seen *as* a problem in prod, but we can circle back.